Контест

svvan45 opened the pull request.

svvan45 committed:
Контест

Nastasia8 commented on k_6.py:
```diff
@@ -0,0 +1,13 @@
+	if t[i+1] > c[i]:
+		print('yes')
+	else:
+		print('no')
```
По-моему, я уже видела такую реализацию)

svvan45 commented:
Я помогал некоторым людям, все задачи я делаю сам

вт, 9 мар. 2021 г., 20:42 Nastasia8 <notifications@github.com>:

> *@Nastasia8* commented on this pull request.
> ------------------------------
>
> In k_6.py
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/38#discussion_r590583566>
> :
>
> > @@ -0,0 +1,13 @@
>
> +n = int(input())
>
> +c = list(map(int, input().split(maxsplit = n)))
>
> +k = int(input())
>
> +p = list(map(int, input().split(maxsplit = k)))
>
> +
>
> +t = [0]*101
>
> +for i in p:
>
> +	t[i] += 1
>
> +for i in range(n):
>
> +	if t[i+1] > c[i]:
>
> +		print('yes')
>
> +	else:
>
> +		print('no')
>
>
> По-моему, я уже видела такую реализацию)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Nastasia8/AaDS_1_185_2021/pull/38#pullrequestreview-607687494>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARAEOFEHFYGQILFB6JYH3JLTCZMXDANCNFSM4Y34YGNA>
> .
>
